### PR TITLE
Rename queue for message [SFEQS-957]

### PR DESCRIPTION
**Pull request by grausof**
This PR rename the queue for message from "waiting_for_signature" to "waiting_for_message"!

### CODE REVIEW

File: SendSignatureRequestMessage/function.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
       "name": "sendSignatureRequestMessage",
       "type": "queueTrigger",
       "direction": "in",
-      "queueName": "waiting-for-signature",
+      "queueName": "waiting-for-message",
```
Comment (gunzip) [inline SendSignatureRequestMessage/function.json]: you may use the environment variable: `%QUEUE_STORAGE_NAME_FOR_WAITING_MESSAGE%`

Comment (grausof) [reply SendSignatureRequestMessage/function.json]: you are right, thanks!

Comment (grausof) [reply SendSignatureRequestMessage/function.json]: it is possible to directly use the environment variable with the syntax `%%`, so I entered `%WaitingMessageQueueName%`!

File: SendSignatureRequestMessage/function.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
       "name": "sendSignatureRequestMessage",
       "type": "queueTrigger",
       "direction": "in",
-      "queueName": "waiting-for-signature",
+      "queueName": "%WaitingMessageQueueName%",
```
Comment (gunzip) [inline SendSignatureRequestMessage/function.json:7]: I think it's %QUEUE_STORAGE_NAME_FOR_WAITING_MESSAGE% (local settings isnt' read in production)

Comment (grausof) [reply SendSignatureRequestMessage/function.json:7]: `QUEUE_STORAGE_NAME_FOR_WAITING_MESSAGE` is only a placeholder for local settings. The real env name is `WaitingMessageQueueName`. https://github.com/pagopa/io-functions-sign/blob/187b304741b898e0589b54dfc3fb777b41c64051/src/infra/azure/storage/webjobs-config.ts#L25

Comment (gunzip) [reply SendSignatureRequestMessage/function.json:7]: Ok!

**Review by gunzip — APPROVED** [review]

**Pull request merged**